content/pages/sprint/index.rst

```diff
@@ -13,6 +13,8 @@ hosted by sciwork.
 Scisprint 2021
 ==============
 
+* `Scisprint 2021 Mid August: Python ZH TW Doc <{filename}2021/08mid-pycht.rst>`__
+* `Scisprint 2021 August <{filename}2021/08.rst>`__
```

Comment:
It's strange the line was not in your base, but it's ok.

Reply:
Probably I dropped  that commit when rebasing. Let me check for it.